src/monodroid/jni/xamarin-app.h

```diff
@@ -21,6 +21,7 @@ struct ApplicationConfig
 	bool uses_assembly_preload;
 	bool is_a_bundled_app;
 	bool broken_exception_transitions;
+	uint8_t bound_exception_type;
```

Comment:
Should we care about struct packing & padding?  (Probably should have asked this earlier...)

Reply:
I don't think we need to